src/components/seeds/database/Filters.tsx

```diff
@@ -305,7 +305,7 @@ function getSearchTermFilter(searchCols: DatabaseColumn[], searchTerm: string):
     children: searchCols.map((col) => ({
       operation: 'field',
       field: col.key,
-      type: 'Fuzzy',
+      type: col.key === 'accessionNumber' ? 'ExactOrFuzzy' : 'Fuzzy',
```

Comment:
for future consideration.. the column definitions can set the search type (defaulting to Fuzzy if undefined)

Reply:
Hmm, i didn't see how to define this. There wasn't a field on the DatabaseColumn type

Reply:
Correct, hence 'future consideration'.. it was just a thought :)
Maybe if/when we end up special casing any more keys, we could lift the search type semantics into the column def.

Reply:
Ah, okay I see what you meant.